### ISSUE TITLE
Upgrade to pytorch 1.2.0

### ISSUE THREAD

**Issue by dmtrs** [wekaco/samplernn-pytorch#22]
- [Cycle learning rate](https://pytorch.org/docs/1.2.0/optim.html?highlight=cycle%20learning%20rate)

**Comment by dmtrs** [comment]
Latest master at this point is  pytorch==1.5.1 torchvision==0.6.1 cudatoolkit=10.
